### DISCUSSION TITLE
How can React Native Webview enable automatic playback of videos,

### DISCUSSION THREAD

**Discussion by ShaoGongBra** [microsoft/react-native-windows#12488] [Q&A] [answered]
The allowsProtectedMedia property does not take effect on Windows without muting

**Comment by chrisglein** [comment]
The `allowsProtectedMedia` property on what?
Are you using an existing module for video playback? For example [`react-native-video`](https://github.com/react-native-video/react-native-video)?

**Comment by ShaoGongBra** [comment, marked as answer]
It's react-native-webview, and I've found the answer here

https://github.com/MicrosoftEdge/WebView2Feedback/discussions/4214
